src/lib/core/CHIPConfig.h

```diff
@@ -1771,6 +1771,22 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MAX_ATTRIBUTE_STORE_ELEMENT_SIZE 1003
 #endif // CHIP_CONFIG_MAX_ATTRIBUTE_STORE_ELEMENT_SIZE
 
+/*
+ * @def CHIP_CONFIG_MAX_OPERATIONAL_NETWORKS
+ *
+ * @brief Maximum number of operational networks that the device may
+ *        advertise in parallel.
+ *
+ *        Value will be used for internal CHIP MDNS implementations but may
+ *        be ignored for platform implementations where the platform implements
+ *        its own memory management for DNSSD advertisement.
+ *
+ *        Set to 0 to mean "unlimited/use heap".
+ */
+#ifndef CHIP_CONFIG_MAX_OPERATIONAL_NETWORKS
+#define CHIP_CONFIG_MAX_OPERATIONAL_NETWORKS 5
```

Comment:
How is that different from `CHIP_CONFIG_MAX_FABRICS`?

Reply:
Well beside the obvious "I did not know about MAX_FABRICS"  this does support 0 for using heap for operational network advertisement.

However I also agree this is duplicating (and numbers seem to differ ... I was told by spec this number should be 5 however I see a lot of MAX_FABRICS being set to 4 on platforms) so maybe just a "USE_HEAP" would be a more correct flag. Looking into it.